server/controllers/inventory/index.js

```diff
@@ -88,6 +93,98 @@ function inventoryLog(req, res, next) {
   }).catch(next);
 }
 
+// get inventory log as excel
+// GET /inventory/download/log/:uuid?rendere=xlsx?lang=fr
+
+async function logDownLoad(req, res, next) {
+  try {
+    const { lang } = req.query;
+    const rows = await core.inventoryLog(req.params.uuid);
+    // inventory columns
+    const cols = {
+      code : 'FORM.LABELS.CODE',
+      consumable : 'FORM.LABELS.CONSUMABLE',
+      default_quantity : 'FORM.LABELS.DEFAULT_QUANTITY',
+      group_uuid : 'FORM.LABELS.GROUP',
+      inventoryGroup : 'FORM.LABELS.GROUP',
+      label : 'FORM.LABELS.LABEL',
+      text : 'FORM.LABELS.LABEL',
+      note : 'FORM.INFO.NOTE',
+      price : 'FORM.LABELS.UNIT_PRICE',
+      sellable : 'INVENTORY.SELLABLE',
+      type_id : 'FORM.LABELS.TYPE',
+      inventoryType : 'FORM.LABELS.TYPE',
+      unit_id : 'FORM.LABELS.UNIT',
+      inventoryUnit : 'FORM.LABELS.UNIT',
+      unit_volume : 'FORM.LABELS.VOLUME',
+      unit_weight : 'FORM.LABELS.WEIGHT',
+    };
+
+    const dictionary = util.loadDictionary(lang);
+
+    const inventory = await core.getItemsMetadata({ uuid : req.params.uuid });
+
+    const lines = [
+      { column1 : '', column2 : '', column3 : '' },
+    ];
+
+    lines.push({
+      column1 : _.get(dictionary, 'FORM.LABELS.INVENTORY'),
+      column2 : inventory[0].label || '',
```

Comment:
Is there ever a case where `inventory[0].label` won't be defined?  If so, what happens?